doc-content/enterprise-only/vehicle-route-planning/run-locally-noninteractive-proc.adoc

```diff
@@ -0,0 +1,21 @@
+[id='run-locally-noninteractive-proc_{context}']
+
+= Run the OptaWeb Vehicle Routing runLocally.sh script in non-interactive mode
+
+Use the non-interactive mode to specify an existing region and start OptaWeb Vehicle Routing with a single command.
+This is useful for switching between regions quickly or when doing a demo.
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* The OSM file for the region that you want to use has been downloaded. For information about downloading OSM files, see xref:run-locally-interactive-proc_{context}[].
+* Internet access is available.
+
+
+.Procedure
+. Change directory to the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaweb-vehicle-routing/sources` directory.
+. Execute the following command where `<REGION>` is a two letter country code for a region that you previously downloaded. For a list of country codes, see https://www.iso.org/iso-3166-country-codes.html[_ISO 3166 Country Codes_].
++
+[source]
+----
+./runLocally.sh <REGION>.
```

Comment:
The script argument must be an OSM file name, not a country code.
so for example ./runLocally.sh belgium-latest

Reply:
`<REGION>` is correct here but its definition is wrong. `<OSM_FILE_NAME>` is a viable alternative. Maybe it's less confusing.

Reply:
Updated